.eslintrc

```diff
@@ -11,39 +11,36 @@
     "plugin:react-hooks/recommended",
     "plugin:import/recommended",
     "plugin:jsx-a11y/recommended",
-    "prettier"
+    "plugin:tailwindcss/recommended",
+    "prettier",
   ],
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
     "ecmaFeatures": {
-      "jsx": true
+      "jsx": true,
     },
     "ecmaVersion": "latest",
-    "sourceType": "module"
+    "sourceType": "module",
   },
-  "plugins": [
-    "react",
-    "@typescript-eslint",
-    "react-hooks",
-    "import",
-    "jsx-a11y",
-    "prettier"
-  ],
+  "plugins": ["react", "@typescript-eslint", "react-hooks", "import", "jsx-a11y", "prettier", "tailwindcss"],
```

Comment:
`prettier` should be last import, because of point 2:
https://github.com/prettier/eslint-config-prettier?tab=readme-ov-file#installation

Reply:
![image](https://github.com/user-attachments/assets/941f45ad-fcf2-4fee-aef0-687b6f52c317)
I have made the requested changes and tried to resolve the merge conflicts please review